logbook-spring-boot-autoconfigure/src/main/java/org/zalando/logbook/autoconfigure/LogbookProperties.java

```diff
@@ -27,6 +27,8 @@ public static class Obfuscate {
         private final List<String> headers = new ArrayList<>();
         private final List<String> parameters = new ArrayList<>();
         private final List<String> paths = new ArrayList<>();
+        private final List<String> jsonBodyFields = new ArrayList<>();
+        private final String replacement = "XXX";
```

Comment:
I wrote a simple test for this, and it fails to run due to `No setter found for property: replacement`.

The test looks like this, but I'm sure you can improve it:

```java
@LogbookTest(properties = "logbook.obfuscate.replacement = ZZZ")
class ObfuscateReplacementTest {

    @Autowired
    private Logbook logbook;

    @Test
    void shouldInitializeLogbook() {
        assertThat(logbook).isNotNull();
    }

}
```

Reply:
Thank you for catching this! I'll add the test.

Reply:
added in https://github.com/zalando/logbook/pull/1592/commits/bb40c742faaaa0ff2980d1fb07da391dcee85dc8